app/src/main/cpp/skyline/common/languages.h

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <frozen/unordered_map.h>
+#include <common.h>
+#include <common/macros.h>
+
+namespace skyline {
+    namespace constant {
+        constexpr size_t OldLanguageCodeListSize{15}; //!< The size of the pre 4.0.0 language code list
+        constexpr size_t NewLanguageCodeListSize{18}; //!< The size of the post 10.1.0 language code list (was 17 between 4.0.0 - 10.1.0)
+    }
+
+    namespace languages {
+        /**
+         * @brief The list of all languages. Entries parameters are language, language code, system language index, application language index and map which holds a macro used for filtering some application languages that don't have a direct corresponding system language
+         * @example #define LANG_ENTRY(lang, code, sysIndex, appIndex, map) func(lang, code, sysIndex, appIndex, map) <br> LANGUAGES <br> #undef LANG_ENTRY
```

Comment:
I don't really like this, using html tags for comments seems silly

Reply:
I know we usually don't use this but that's the only way to get it formatted properly in the IDE tooltip

Reply:
Riight... Do you actually use the ide tooltip?

Reply:
yes

Reply:
Kinda quirky ngl